src/azure-cli/azure/cli/command_modules/storage/azcopy/util.py

```diff
@@ -53,10 +53,13 @@ def install_azcopy(self, install_location):
         else:
             raise CLIError('Azcopy ({}) does not exist.'.format(self.system))
         try:
-            os.chmod(install_dir, os.stat(install_dir).st_mode | stat.S_IWUSR)
             _urlretrieve(file_url, install_location)
             os.chmod(install_location,
                      os.stat(install_location).st_mode | stat.S_IXUSR | stat.S_IXGRP | stat.S_IXOTH)
+        except PermissionError as err:
+            raise CLIError(f'Permission error while attempting to download azcopy to {install_dir}. '
+                           f'You could install the specified azcopy version {AZCOPY_VERSION} manually or '
+                           f'grant write permission and retry. ({err})')
```

Comment:
`PermissionError` is thrown by https://github.com/Azure/azure-cli/blob/c0276baf0208e1ee2f82ae5ed3bdce4de7275748/src/azure-cli/azure/cli/command_modules/storage/azcopy/util.py#L213-L214

Guess it's better to capture it here?